app/javascript/store/modules/profile/actions.js

```diff
@@ -25,6 +30,16 @@ export default {
     dispatch(COMPUTE_RECOMMENDATIONS, { teamId, githubUserLogin });
   },
 
+  [PROCESS_NEW_ORGANIZATION](
+    { commit }, { organizationId, teams }) {
+    let teamIndex = 0;
```

Comment:
'teamIndex' is never reassigned. Use 'const' instead  prefer-const<br>'teamIndex' is assigned a value but never used        no-unused-vars